tests/StreamWrapperTest.php

```diff
@@ -119,21 +119,8 @@ public function testUrlStat(): void
 
         $stBlksize  = defined('PHP_WINDOWS_VERSION_BUILD') ? -1 : 0;
 
-        self::assertEquals(
+        self::assertSame(
```

Comment:
should stay assertEquals as the order of the element should not matter here

Reply:
Reverted https://github.com/guzzle/psr7/pull/340/commits/14ca244baa35c4c98f159e03f220e56490300d40